life_expectancy/cleaning.py

```diff
@@ -35,21 +46,16 @@ def clean_data(country):
 
```

Comment:
**suggestion**: Use 
`dataframe["value"] = dataframe["value"].str.replace(r"[^0-9.]+", "", regex=True)`

This makes ue of pandas' vectorization